Sources/Services/ContainerNetworkService/NetworkService.swift

```diff
@@ -101,7 +101,7 @@ public actor NetworkService: Sendable {
         }
 
         let hostname = try message.hostname()
-        let index = try await allocator.lookup(hostname: hostname)
+        let index = try allocator.lookup(hostname: hostname)
```

Comment:
Because AttachmentAllocator is an actor, callers still need to access every method asynchronously. The removal of the `async` from the `func lookup(hostname: String)` definition does not change this, it's solely to mark that that method does not call anything async inside of itself.